pkg/inventory/web/client.go

```diff
@@ -451,6 +467,12 @@ func (r *Watch) Repair() (err error) {
 //
 // End the watch.
 func (r *Watch) End() {
+	r.reader.log.V(3).Info("wtach end requested.")
+	_ = r.reader.webSocket.WriteJSON(
+		Event{
+			Action: libmodel.End,
+		})
+	time.Sleep(50 * time.Millisecond)
 	r.reader.Terminate()
```

Comment:
Send an `End` _event_ to the server before we just kill the reader and close the webscoket.  This will allow the server (WatchWriter) to propagate the watch end (to the model layer) and shutdown normally.